doc/rst/technotes/gpu.rst

```diff
@@ -184,8 +184,8 @@ code for and interacts with GPUs. These variables include:
   in the future.
 * ``CHPL_GPU_NO_CPU_MODE_WARNING`` - this variable is relevant when using the
   `CPU as Device mode`_ and if set causes it so that uses of
-  :proc:`~GPU.assertOnGpu` to not generate a warning at
-  execution time. Alternatively, this behavior can be enabled by passing
+  :proc:`~GPU.assertOnGpu` and the ``@assertOnGpu`` attribute to not generate a
```

Comment:
I wouldn't mind if you dropped the proc version from the technote completely. But I'll leave the final call to you. We should remove it from the module and the technote by 1.33 for sure.

Reply:
I think I want to keep it for the time being, but I agree with removing it for 1.33.